lms/djangoapps/ccx/tests/test_field_override_performance.py

```diff
@@ -125,15 +126,12 @@ def setup_course(self, size, enable_ccx, view_as_ccx):
             self.student,
             course_key
         )
+        return CourseKey.from_string(unicode(course_key))
```

Comment:
@ormsbee Here's the line of code that I needed to add in order to workaround the `unicode`/`str` issues with opaque keys.  Otherwise, the key comparison wasn't working when looking for the block-key in the block_structure.